nncf/quantization/algorithms/accuracy_control/openvino_backend.py

```diff
@@ -38,6 +39,10 @@ class OVAccuracyControlAlgoBackend(AccuracyControlAlgoBackend):
 
     # Metatypes
 
+    @staticmethod
+    def get_weighted_metatypes() -> List[OVOpMetatype]:
+        return [ov_metatypes.OVMatMulMetatype, ov_metatypes.OVEmbeddingMetatype]
```

Comment:
Why the list of the weighted metatypes is limited and not correlated with the `OPERATIONS_WITH_WEIGHTS`?

Reply:
I used operation metatypes to which weight compression is applied.

https://github.com/openvinotoolkit/nncf/blob/5f2c20e35cf3c1aa284275c7d4fd7ed7593a6522/nncf/quantization/algorithms/weight_compression/openvino_backend.py#L38-L39

Reply:
Why do we focus on the `weight compression` and not the `quantize` algorithm?

Reply:
As far as I understand, the main idea behind the `backup_mode` parameter is to revert operations not only to their original precision but also to a representation with INT8 compressed weights. Therefore, we need to select a subset of operations with weights for which we want to preserve INT8 compressed weights. I followed the logic of the weights compression algorithm, wherein we compress weights only for `MatMul` and `Embedding` operations.

Reply:
I don't get it.
Anyway, I'll leave the final decision to @alexsu52 and @AlexKoff88

Reply:
Good catch! I hope we can apply this logic to all quantized operations not only for operation which support for weights compression, if it does not lead to performance degradation.

Reply:
I reverted half of all convolutions in MobileNetV2 using both `RestoreMode.ACTIVATIONS_AND_WEIGHTS` and `RestoreMode.ONLY_ACTIVATIONS`. There was no difference in performance between the resulting models, so I believe we can specify `OPERATIONS_WITH_WEIGHTS` here.